lib/migrate/uglifyJsPlugin/__snapshots__/uglifyJsPlugin.test.js.snap

```diff
@@ -2,36 +2,118 @@
 
 exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-0" data 1`] = `
 "module.exports = {
-    plugins: [
-        new webpack.optimize.UglifyJsPlugin({
-            sourceMap: true
-        })
-    ]
+    optimization: {
+        minimize: true
+    }
 }
 "
 `;
 
 exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-1" data 1`] = `
-"module.exports = {
+"const UglifyJsPlugin  = require('uglifyjs-webpack-plugin');
+module.exports = {
     devtool: \\"source-map\\",
-    plugins: [
-        new webpack.optimize.UglifyJsPlugin({
-            sourceMap: true
-        })
-    ]
+    optimization: {
+        minimize: true,
+
+        minimizer: [new UglifyJsPlugin({
+            sourceMap: true,
+            compress: {}
+        })]
+    }
 }
 "
 `;
 
 exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-2" data 1`] = `
+"const Uglify  = require('uglifyjs-webpack-plugin');
+module.exports = {
+    devtool: \\"source-map\\",
+    optimization: {
+        minimize: true,
+
+        minimizer: [new Uglify({
+            sourceMap: true,
+            compress: {}
+        })]
+    }
+}
+"
+`;
+
+exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-3" data 1`] = `
 "module.exports = {
-    devtool: \\"cheap-source-map\\",
-    plugins: [
-        new webpack.optimize.UglifyJsPlugin({
-            compress: {},
+    devtool: 'eval',
+
+    entry: [
+		'./src/index'
+	],
+
+    output: {
+		path: path.join(__dirname, 'dist'),
+		filename: 'index.js'
+	},
+
+    module: {
+		loaders: [{
+			test: /.js$/,
+			loaders: ['babel'],
+			include: path.join(__dirname, 'src')
+		}]
+	},
+
+    resolve: {
+		root: path.resolve('/src'),
+		modules: ['node_modules']
+	},
+
+    plugins: [new webpack.optimize.OccurrenceOrderPlugin()],
+    debug: true,
+
+    optimization: {
+        minimize: true
+    }
+};
+"
+`;
+
+exports[`uglifyJsPlugin transforms correctly using "uglifyJsPlugin-4" data 1`] = `
+"const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+module.exports = {
+    devtool: 'eval',
+
+    entry: [
+		'./src/index'
+	],
+
+    output: {
+		path: path.join(__dirname, 'dist'),
+		filename: 'index.js'
+	},
+
+    module: {
+		loaders: [{
+			test: /.js$/,
+			loaders: ['babel'],
+			include: path.join(__dirname, 'src')
+		}]
+	},
+
+    resolve: {
+		root: path.resolve('/src'),
+		modules: ['node_modules']
+	},
+
+    plugins: [new webpack.optimize.OccurrenceOrderPlugin()],
+    debug: true,
+
+    optimization: {
+        minimize: true,
+
+        minimizer: [new UglifyJsPlugin({
```

Comment:
Is this correct? Could you link me to the docs on this?

Reply:
https://github.com/webpack/webpack/blob/master/test/configCases/plugins/uglifyjs-plugin/webpack.config.js#L20

Reply:
Oki, good stuff